meson.build

```diff
@@ -109,7 +109,7 @@ endif
 
 wayland_scanner = find_program(wayland_scanner_candidates, native: true)
 
-r = run_command(wayland_scanner, '--version')
+r = run_command(wayland_scanner, '--version', check: false)
 assert(r.returncode() == 0,
 	'''Cannot execute wayland-scanner:
 	''' + r.stderr())
```

Comment:
Why `assert` instead of just having `check` `true`?

Reply:
Good point, I just thought “it's checked right below, so let's `check: false`” but you are right.